docs/troubleshoot/troubleshoot-apiml.md

```diff
@@ -31,9 +31,9 @@ its performance and create large log files that consume a large volume of disk s
 
 5. (Optional) Reproduce a bug that causes issues and review debug messages. If you are unable to resolve the issue, create an issue [here](https://github.com/zowe/api-layer/issues/).     
 
-6. Disable the debug mode. Modify the line which contains the `spring.profiles.include` parameter back to default:
+6. Disable the debug mode. Modify the line which contains the `LOG_LEVEL=` parameter back to default:
     ```
```

Comment:
Find the `LOG_LEVEL=` parameter, and change its current value to the default `LOG_LEVEL=` one.